packages/rule-https-only/README.md

```diff
@@ -0,0 +1,32 @@
+# https-only (`https-only`)
+
+Description for https-only
+
+## Why is this important?
+
+Explain why this rule is important for your users
+
+## What does the rule check?
+
+A bit more detail of what the rule does.
+
+### Examples that **trigger** the rule
+
+A list of code examples that will fail this rule.
+It's good to put some edge cases in here.
+
+### Examples that **pass** the rule
+
+A list of code examples that will pass this rule.
+It's good to put some edge cases in here.
+
+## Can the rule be configured?
+
+If this rule allows some configuration, please put the format and
+options for the user.
+
+## Further Reading
+
+What can the user read to know more about this subject?
```

Comment:
When you write the docs, add this https://doesmysiteneedhttps.com/ (and also feel free to get some inspiration from it).

Reply:
ok!